src/app/content/highlights/components/Card.spec.tsx

```diff
@@ -36,13 +35,19 @@ describe('Card', () => {
   let dispatch: jest.SpyInstance;
   let highlight: ReturnType<typeof createMockHighlight>;
   let highlightData: ReturnType<ReturnType<typeof createMockHighlight>['serialize']>['data'];
+  let cardProps: Partial<CardProps>;
 
   beforeEach(() => {
     store = createTestStore();
     highlight = createMockHighlight('asdf');
```

Comment:
You could define highlight as `Highlight` and cast it where `createMockHighlight` is called rather than in every test